tools/scripts/icu-file-utf8-check.py

```diff
@@ -32,6 +34,9 @@
 import getopt
 
 
+# List of directories to check for UTF-8 and BOM. Extend as needed.
+icu_directories_to_be_scanned = ["icu4c/source", "icu4j"]
```

Comment:
Does it work if we scan all files? Or at least "tools"? Or is there too much amiss right now?